.devcontainer/devcontainer.json

```
@@ -0,0 +1,11 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/php-mariadb
+{
+	"name": "GLPI docker devcontainer",
+	"dockerComposeFile": "docker-compose.yml",
+	"service": "php",
+	"workspaceFolder": "/var/www/",
```

It seems to me that the norm is to place it in a subfolder of `/var/www`.
```suggestion
	"workspaceFolder": "/var/www/glpi",
```

---

One devcontainer/codespace per project, so I guess it's not usefull